.travis.yml

```diff
@@ -221,325 +259,359 @@ matrix:
           repo: pantsbuild/pants
 
     # Deploy Pex Unstable to s3
-    - name: "Deploy Pants PEX Unstable"
+    - name: "Deploy unstable multiplatform pants.pex"
```

Comment:
Idle question for the future (definitely not for now): Would it make sense to merge the stable and unstable pex publish jobs? The difference between the two pexes is basically a handful of strings and hashes, right?

Reply:
I don't actually know. I was out of the loop when this was originally implemented.